src/verify.ts

```diff
@@ -83,7 +83,7 @@ export const registryVerifier: Verifier<
       const documentData = getData(document);
       return documentData.proof.method === v3.Method.DocumentStore;
     }
-    const documentData = getData(document);
+    const documentData = getData(document as WrappedDocument<v2.OpenAttestationDocument>);
```

Comment:
why ?

Reply:
there's build error: https://app.circleci.com/pipelines/github/OpenCerts/verify/66/workflows/e23b03ce-4272-4df6-931e-887735406e1e/jobs/64

Reply:
ok but that's not good. The purpose of using DeepPartial is to make sure we do the correct checking here, however by casting to `WrappedDocument<v2.OpenAttestationDocument>` we just get rid of that purpose.

I'm not sure how to fix this now

Reply:
this is the best I could come up with https://github.com/Open-Attestation/oa-verify/blob/master/src/verifiers/documentStoreStatus/openAttestationEthereumDocumentStoreStatus.ts#L52

let's use sth like this for the moment